app/modules/entities/components/lib/works_browser_helpers.js

```diff
@@ -49,6 +52,15 @@ const setWorkFacets = ({ facets, valuesUris, contextProperties }) => work => {
   }
 }
 
+const removeFacetsWithNoKnownValue = facets => {
+  Object.keys(facets).forEach(prop => {
+    const facet = facets[prop]
+    if (notOnlyUnknownKey(facet)) delete facets[prop]
+  })
```

Comment:
very much non-blocking: this days I prefer this syntax
```suggestion
  for (const [ prop, facet ] of Object.entries(facets)) {
    if (notOnlyUnknownKey(facet)) delete facets[prop]
  }
```

Reply:
yeah more obscure for JS newcomers in my opinion (`.forEach` and `.keys` are quite basic compared to array destructuring and `.entries`)
but done anyway in 81dbda77e

Reply:
Come on, it's not some dirty hack, it's a very well documented standard function, now very well established in the JS ecosystem, and giving the possibility to map/filter on non-array objects: that should be on every JS dev toolbox! If you consider it to be obscure, I'm happy to bring some light to it ;)